libs/shared/ui-components/src/Video/VideoThumbnail.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+
+import clsx from 'clsx';
+
+import Picture from '../Picture/Picture';
+import { TVideoThumbnailProps } from './types';
+
+const styles = {
+  button: `
+    absolute top-[50%] left-[50%] translate-x-[-50%] translate-y-[-50%] w-[6rem] h-[6rem] rounded-full z-1000 bg-violet transition-[.4s] group-hover:scale-[1.1]
+    before:content-[''] before:absolute before:top-[1.5rem] before:left-[2.2rem] before:w-0 before:h-0 before:border-t-[1.5rem] 
+    before:border-b-[1.5rem] before:border-l-[2.4rem] before:border-r-none before:border-transparent before:border-l-white
+  `,
+  wrapper: `
+    group overflow-hidden absolute top-0 left-0 z-10 w-full h-full transition-[.3s] cursor-pointer transition-[.4s]
+    hover:brightness-110
+  `,
+};
+
+export const VideoThumbnail = ({ src, alt, onClick }: TVideoThumbnailProps) => {
+  const [hidden, setHidden] = useState(false);
+
+  const wrapperStyles = clsx(styles.wrapper, hidden && 'hidden');
+
+  const handleClick = () => {
+    setHidden(true);
+    onClick();
+  };
+
+  return (
+    <div className={wrapperStyles} onClick={handleClick} aria-hidden={true}>
```

Comment:
```suggestion
    <div className={wrapperStyles} onClick={handleClick} aria-hidden={hidden}>
```

Reply:
OK, to be changed

Reply:
@gabalafou your proposal is causing [this kind of error](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/93f78856655696a55309440593e0948c6fb96134/docs/rules/click-events-have-key-events.md), so I think we should keep aria-hidden true anyways. changed syntax to proposed by a11y: `aria-hidden="true"`.

Reply:
That would be bad for accessibility. Here's why. I applied this PR. Then I loaded a sample video on a page. Then I visited the page using a screen reader (VoiceOver on Mac). I was unable to find and play the video using only the screen reader. Which makes sense. If you apply `aria-hidden="true"` to an element, you're telling the browser to hide that element from the screen reader. 

There are a couple other accessibility issues:

- The play button needs a label. A person who cannot see has no way of knowing that the button is a play button. The screen reader just says "button."
- There's no way to stop the video. People need to be able to push pause after pushing play.